frontend/webapp/src/pages/screen-recorder.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useRef } from "react";
+
+export default function ScreenRecorder() {
+  const [isRecording, setIsRecording] = useState(false);
+  const [videoUrl, setVideoUrl] = useState(null);
+  const mediaRecorder = useRef(null);
+  const chunks = useRef([]);
+
+  const startRecording = async () => {
```

Comment:
on testing this we dont hear the audio getting recorded, please verify that audio also gets recorded

Reply:
sir it is recording audio only for window

Reply:
this is not resolved,
make sure it is recording audio by default, so we do not have to manually toggle the audio button everttime

Reply:
attach screenshots, short video for showing us what you mean

Reply:
@Swapnilden Did you get a chance to to fix audio ?

Reply:
@janakiramg Yes, the audio issue was addressed

Reply:
I can verify from the demo video that audio is being recorded and played back